dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -126,6 +127,28 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 					if (trails.length === 0) return null;
 
 					const ophanName = ophanComponentId(collection.displayName);
+					const ophanComponentLink = `container-${
+						index + 1
+					} | ${ophanName}`;
+
+					if (collection.collectionType === 'fixed/thrasher') {
+						if (trails[0].snapData === undefined) {
+							return (
+								<>
+									First item of ${collection.displayName}
+									should be a SnapLink
+								</>
+							);
```

Comment:
I spent a while thinking about this! My worry was that I think the front tool will let you place a normal card in a `fixed/thrasher` container instead of a Snap, I don't think we'd ever want to allow that but at the same time I don't want to return a 500 error because one container is misconfigured. It seemed better to fail gracefully this way, but happy to change if people disagree.

Reply:
> My only concern is this might get left in and appear in production

This is a possibility in the current state, and surfacing debug data to readers is undesirable. [@mxdvl did some logging with unwrapHTML](https://github.com/guardian/dotcom-rendering/pull/4036/files). I'm not sure using that logger is the right place, but having any logs in Kibana would be useful.

Or maybe Sentry? Or though I think our Sentry is a little messy.

Reply:
For the logs to Kibana, rather than the logs endpoint, [it’s this commit](https://github.com/guardian/dotcom-rendering/commit/a8274881eb6a758a3732f283a9f0e3f0dcfa3f81)!

Reply:
Kibana sounds promising, I'll take a look at doing that.

I think Sentry is just for client side errors unless I'm mistaken